src/test/webapp/app/usermanagement/research-group/research-group-members/research-group-members.component.spec.ts

```diff
@@ -0,0 +1,228 @@
+  };
+
+  const mockCurrentUser: UserShortDTO = {
+    userId: 'current-user',
+    firstName: 'Current',
+    lastName: 'User',
+    roles: [UserShortDTO.RolesEnum.Admin],
+  };
+
+  const mockPageResponse: PageResponseDTOUserShortDTO = {
+    content: [mockUserData, mockCurrentUser],
+    totalElements: 2,
+  };
+
+  beforeEach(async () => {
+    mockResearchGroupService = {
+      getResearchGroupMembers: vi.fn(),
+      removeMemberFromResearchGroup: vi.fn(),
+    };
+
+    mockToastService = createToastServiceMock();
+
+    await TestBed.configureTestingModule({
+      imports: [ResearchGroupMembersComponent],
+      providers: [
+        { provide: ResearchGroupResourceApiService, useValue: mockResearchGroupService },
+        { provide: AccountService, useValue: { userId: 'current-user' } },
+        provideToastServiceMock(mockToastService),
+        provideTranslateMock(),
+      ],
+    }).compileComponents();
+
+    fixture = TestBed.createComponent(ResearchGroupMembersComponent);
+    component = fixture.componentInstance;
+  });
+
+  afterEach(() => {
+    vi.clearAllMocks();
+  });
+
+  it('should create', () => {
+    expect(component).toBeTruthy();
+  });
+
+  it('should initialize with default values', () => {
+    expect(component.members()).toEqual([]);
+    expect(component.pageNumber()).toBe(0);
+    expect(component.pageSize()).toBe(10);
+    expect(component.total()).toBe(0);
+  });
+
+  it('should have correct columns configuration', () => {
+    fixture.detectChanges();
+
+    const columns = component.columns();
+    expect(columns).toHaveLength(4);
+    expect(columns[0].field).toBe('name');
+    expect(columns[0].header).toBe('researchGroup.members.tableColumns.name');
+    expect(columns[1].field).toBe('email');
+    expect(columns[2].field).toBe('role');
+    expect(columns[3].field).toBe('actions');
+  });
+
+  it('should update page number and load members on table emit', () => {
+    const event: TableLazyLoadEvent = {
+      first: 20,
+      rows: 10,
+    };
+    mockResearchGroupService.getResearchGroupMembers.mockReturnValue(of(mockPageResponse));
+
+    component.loadOnTableEmit(event);
+
+    expect(component.pageNumber()).toBe(2); // 20 / 10 = 2
+    expect(mockResearchGroupService.getResearchGroupMembers).toHaveBeenCalledWith(10, 2);
+    expect(mockResearchGroupService.getResearchGroupMembers).toHaveBeenCalledTimes(1);
+  });
+
+  it('should handle undefined first and rows values in table emit with fallback to defaults', () => {
+    const event: TableLazyLoadEvent = {};
+    mockResearchGroupService.getResearchGroupMembers.mockReturnValue(of(mockPageResponse));
+
+    component.loadOnTableEmit(event);
+
+    expect(component.pageNumber()).toBe(0);
+    expect(component.pageSize()).toBe(10);
+    expect(mockResearchGroupService.getResearchGroupMembers).toHaveBeenCalledWith(10, 0);
+    expect(mockResearchGroupService.getResearchGroupMembers).toHaveBeenCalledTimes(1);
+  });
+
```

Comment:
You could add another test to check wether `loadMembers()` is called on init:

```suggestion

  it('should load members automatically via table lazy load on component init', () => {
    mockResearchGroupService.getResearchGroupMembers.mockReturnValue(of(mockPageResponse));

    fixture.detectChanges(); 

    expect(mockResearchGroupService.getResearchGroupMembers).toHaveBeenCalledWith(10, 0);
    expect(mockResearchGroupService.getResearchGroupMembers).toHaveBeenCalledTimes(1);
  });
    
```